OpenSim/Simulation/Model/Appearance.h

```diff
@@ -33,105 +33,160 @@ namespace OpenSim {
 class Body;
 class Model;
 
+/**
+VisualRepresentation is the OpenSim name used across the OpenSim API, it maps to
+SimTK::DecorativeGeometry::Representation.
+*/
```

Comment:
This comment does not say what a `VisualRepresentation` is. Is it useful for the user of the API to know the existence of `SimTK::DecorativeGeometry::Representation`?